licenses.yaml

```diff
@@ -3135,7 +3135,7 @@ name: Apache Parquet
 license_category: binary
 module: extensions/druid-parquet-extensions
 license_name: Apache License version 2.0
-version: 1.10.1
+version: 1.11.0
 libraries:
   - org.apache.parquet: parquet-avro
```

Comment:
Apologies for the churn, but it looks like maybe the updated version adds a new dependency which wasn't previously in the list:

```
Error: found 1 missing licenses. These licenses are reported, but missing in the registry
druid_module: parquet-extensions, groupId: org.apache.parquet, artifactId: parquet-format-structures, version: 1.11.0, license: Apache License version 2.0
```

I think if you add `parquet-format-structures` to this list of `libraries` then it should be ok?

Reply:
No worries @clintropolis . I've also been quite busy lately.